docs/index.rst

```diff
@@ -635,6 +635,58 @@ route:
 **method**    GET, POST
 ============ ================================
 
+"But I don't want all endpointes use same method!" I hear you cry. Worry not!
+You have two options how to create endpoint with different methods.
+
+Let's see the first one: ::
+
+    from flask_classful import route
+
+    class MyResourceView(FlaskView):
+
+        def edit(self, id):
+            ...
+
+        @route("update/<id>", methods=["POST"])
+        def update(self, id):
+            ...
+
+``@route`` is an old friend of ours, but I didn't tell you earlier that it can also set methods, right? Well, now you know. As you can see, ``methods`` can accept multiple values, so you can set multiple methods for one endpoint (but again, but recommanded).
```

Comment:
[...] ~but~ **not** recomm~a~**e**nded [...]

Reply:
oh my, I almost always do this typo. thanks!